models/spleen_ct_segmentation/configs/multi_gpu_evaluate.json

```diff
@@ -0,0 +1,26 @@
+{
+    "device": "$torch.device(f'cuda:{dist.get_rank()}')",
+    "network": {
+        "_target_": "torch.nn.parallel.DistributedDataParallel",
+        "module": "$@network_def.to(@device)",
+        "device_ids": [
+            "@device"
+        ]
+    },
+    "validate#sampler": {
+        "_target_": "DistributedSampler",
+        "dataset": "@validate#dataset",
+        "even_divisible": false,
+        "shuffle": false
+    },
+    "validate#dataloader#sampler": "@validate#sampler",
+    "validate#handlers#1#_disabled_": "$True if dist.get_rank() > 0 else False",
```

Comment:
```suggestion
    "validate#handlers#1#_disabled_": "$dist.get_rank() > 0",
```

Reply:
Updated in the latest commit.

Thanks.